CataclysmWin.cbp

```diff
@@ -7,170 +7,309 @@
 		<Option compiler="gcc" />
 		<Build>
 			<Target title="Release">
-				<Option output="Cataclysm" prefix_auto="1" extension_auto="1" />
+				<Option output="CataclysmWin" prefix_auto="1" extension_auto="1" />
```

Comment:
Marginally irked at being tagged as "CataclysmWin", seems unnecessary to mark as Not Linux.  :-/